shared/src/libs/exceptions/incorrect-password-error/incorrect-password-error.exception.ts

```diff
@@ -0,0 +1,22 @@
+import { HTTPCode } from '#libs/packages/http/http.js';
+import { type ValueOf } from '#libs/types/value-of.type.js';
+
+import { AuthError } from '../exceptions.js';
+
+type Constructor = {
+  message: string;
+  status?: ValueOf<typeof HTTPCode>;
+  cause?: unknown;
+};
+
+class IncorrectPasswordError extends AuthError {
```

Comment:
Is it ok to use it or am I supposed to make InvalidCredentialsClass as it is in the mini-project?

Reply:
I think you need to ask about specification for Sign In in the issue. On some websites you can see different errors: "user not found" or "incorrect password". But there is also websites where you can only see one error like "wrong email or password"